mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -66,17 +66,15 @@ class Rock_ConvOpBase<string mnemonic, list<Type> inputTypes=[F32, F16, BF16], l
                        DeclareOpInterfaceMethods<RockConvInterface>,
                        DeclareOpInterfaceMethods<MemoryEffectsOpInterface>,
                        RockFusionRoot]>{
-    dag commonConvArgs = (ins TensorOrMemRefOf<inputTypes>:$filter,
-                              TensorOrMemRefOf<inputTypes>:$input,
-                              TensorOrMemRefOf<outputTypes>:$output,
-                              OptionalAttr<Rock_GemmFeaturesAttr>:$features,
-                              OptionalAttr<I32Attr>:$derivedBlockSize,
-                              OptionalAttr<I32Attr>:$gridSize,
-                              IndexArrayLength<4>:$padding,
-                              IndexArrayLength<2>:$strides,
-                              IndexArrayLength<2>:$dilations,
-                              OptionalAttr<RockTuningParamAttrInterface>:$params);
```

Comment:
nit: 
better to keep formatting as it is on tablegen files. It is much more readable if each param is on different line

Reply:
Can we create `.clang-format-ignore` and add `*.td` files in there?